.github/workflows/deploy.yml

```diff
@@ -3,6 +3,8 @@ name: Deploy
 on:
   push:
     branches: [master]
+  schedule:
```

Comment:
Víme, jak tohle interaguje s větvemi? Deploy se udělá vždycky po pushnutí do masteru, to je jasný, ale ten naplánovaný trigger jede nad kterou větví? V dokumentaci píšou:

> Scheduled workflows run on the latest commit on the default or base branch.

Což by byla větev `next`? Neměli bysme v té akci `checkout` níže nastavit radši explicitně `ref` na `master`? (Viz [dokumentaci pro checkout](https://github.com/actions/checkout).)

Reply:
Chybička se vloudila, měl by řešit  [bde77d6](https://github.com/cesko-digital/obce/pull/24/commits/bde77d6de29a85d20df2c8afff54f72d089df9d2)